app/models/user.rb

```diff
@@ -79,7 +79,7 @@ def items
   def create_personal_group
```

Comment:
Improvement idea: rename to `create_personal_group!` to indicate that this does really change the database and is not just a read-only method. (AFAICT the usage of a suffix "!" is not generally equal among the rails community, i.e. there is no clear cut standard, when to use it, except for the distinction of make-copy and make-immediately methods, but this seems like a good distinction, since we are on a model).